src/interface/playerSelection/PlayerTile.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+
+
+import RoleIcon from 'common/RoleIcon';
+
+
+class PlayerTile extends React.PureComponent {
+  static propTypes = {
+    player: PropTypes.object.isRequired,
+    selectedPlayer: PropTypes.object,
+    analysisUrl: PropTypes.string.isRequired,
+    handleClick: PropTypes.func.isRequired,
+    history: PropTypes.object.isRequired,
+  };
+
+  timeout = null;
+
+  render() {
+    const { player, selectedPlayer, analysisUrl, handleClick, history } = this.props;
+    return (
+      <span
+        key={player.guid}
+        onClick={() => { this.timeout = setTimeout(function(){ handleClick(); }, 200); }}
```

Comment:
Do browsers not have this behavior by default? I remember something like that being the case.

Reply:
It wasn't working for me at any rate. When I was double clicking without the timeout, I'd find that the player would first be clicked and show the right panel, and then the position of the cards would change, and the second click would be on a different person and analyze that person.

Adding the timeout fixed the issue